electron-builder.json

```diff
@@ -14,23 +14,76 @@
   "afterSign": "electron-builder-notarize",
   "artifactName": "${productName}-${os}-${arch}.${ext}",
   "compression": "normal",
+  "asarUnpack": [
+    "node_modules/keytar",
+    "node_modules/sqlite3",
+    "node_modules/cpu-features"
```

Comment:
just let you know that cpu-features is not mandatory and might be removed by --ignore-optional during yarn install. Just in case if there some issues and difficulties with this

Reply:
ok, I'll remove